src/folderlistmodel.cpp

```diff
@@ -364,6 +432,23 @@ FolderListModel::FolderStandardType FolderListModel::folderTypeFromId(const QMai
     return NormalFolder;
 }
 
+bool FolderListModel::isStandardFolder(const QMailFolderId &id) const
+{
+    FolderStandardType folderType = folderTypeFromId(id);
+    return folderType == InboxFolder || folderType == DraftsFolder
```

Comment:
Yes, since this is optional and not used atm(no functionality to mark as spam or similar) we are not matching or adding it if does not exist.